src/transactions/utils/getTransactionBytes.js

```diff
@@ -244,11 +244,9 @@ export default function getTransactionBytes(transaction) {
 		: Buffer.alloc(0);
 
 	const transactionRecipientID = transaction.recipientId
-		? Buffer.from(
-				cryptoModule.bigNumberToBuffer(
-					transaction.recipientId.slice(0, -1),
-					BYTESIZES.RECIPIENT_ID,
-				),
+		? cryptoModule.bigNumberToBuffer(
+				transaction.recipientId.slice(0, -1),
```

Comment:
Weird indentation here. Did prettier do this?

Reply:
Yes, this is coming from prettier